extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/V2CommandsPrivilegesSuite.scala

```diff
@@ -515,6 +515,26 @@ abstract class V2CommandsPrivilegesSuite extends PrivilegesBuilderSuite {
     assert(accessType === AccessType.UPDATE)
   }
 
+  test("DescribeTable") {
+    assume(!isSparkV2)
+
+    val plan = executePlan(s"DESCRIBE TABLE $catalogTable").analyzed
+    val (inputs, outputs, operationType) = PrivilegesBuilder.build(plan, spark)
+    assert(operationType === DESCTABLE)
+    assert(inputs.isEmpty)
+    assert(outputs.size === 1)
+    val po = outputs.head
```

Comment:
I think the table should be treated as input table rather than an output one as which it is reading info from。

Reply:
thanks for the review, ok i change to have the same behavior than DescribeTableCommand